demo/playground/examples/tests/test_mediator_ping_agents.py

```diff
@@ -99,23 +99,23 @@ def initialize_mediation(agent: Agent, invitation):
 
 @pytest.fixture(scope="session")
 def faber_mediator(faber, mediation_invite):
-    logger.info(f"faber_mediator...")
+    logger.info("faber_mediator...")
```

Comment:
Although these string constants do not need the f"..." prefix, I think the author of this code used that convention to be consistent, knowing that very often, log statements would end up having parameters inserted: f"...{param}...". By putting the prefix on the string, it is guaranteed to be safe to insert the parameter. Without that prefix, every insert of a parameter requires the developer to analyze whether the prefix is needed. If the developer makes a mistake, the line has a bug, but the bug isn't detected unless the logged event actually happens, leaving the code a bit vulnerable.

For that reason, I'm not sure it's such a great idea to remove the prefix. But I don't feel super strongly about it. I'm just mentioning it as something to think about.

Reply:
agree, restored.